modin/pandas/test/test_groupby.py

```diff
@@ -2182,6 +2182,26 @@ def test_mean_with_datetime(by_func):
     eval_general(modin_df, pandas_df, lambda df: df.groupby(by=by_func(df)).mean())
 
 
+def test_groupby_mad_warn():
+    modin_df, pandas_df = create_test_dfs(test_groupby_data)
+    md_grp = modin_df.groupby(by=modin_df.columns[0])
+    pd_grp = pandas_df.groupby(by=pandas_df.columns[0])
+
+    msg = "The 'mad' method is deprecated and will be removed in a future version."
+    for grp_obj in (md_grp, pd_grp):
+        with pytest.warns(FutureWarning, match=msg):
```

Comment:
These warnings come from worker processes, we don't need to separately create it in the main process.